components/stacks-network/src/chains_coordinator.rs

```diff
@@ -6,7 +6,6 @@ use crate::event::Status;
 use crate::orchestrator::ServicesMapHosts;
 
 use base58::FromBase58;
-use chainhook_sdk::chainhooks::types::ChainhookConfig;
```

Comment:
We'll want to revert the changes in this file so that we still register hooks with the devnet.